### DISCUSSION TITLE
Mu needs a Big Sur compliant icon for Mac

### DISCUSSION THREAD

**Discussion by ajsirota** [mu-editor/mu#1653] [Ideas & New Features]
Currently, Mu doesn't fit in on the Big Sur dock, with the new icon style.
<img width="248" alt="image" src="https://user-images.githubusercontent.com/67526318/122852621-c9257e80-d2c5-11eb-8839-e0860507f62d.png">

A new icon for Big Sur would be great.

**Comment by ntoll** [comment]
@ajs256 do you have dimensions for the preferred icon style size..? Links to docs for guidance about this..? Or could this just be a case of square icons have the same dimensions as our icon, but just look smaller because they're square..?!?!? ;-)

**Reply by ajsirota** [reply]
I *believe* you make it in multiple resolutions and package them into an ICNS file. [Here](https://developer.apple.com/design/human-interface-guidelines/macos/icons-and-images/app-icon/) are Apple's docs.

**Reply by MohanMaker** [reply]
Yes, it would be great if Mu updated to have a macOS Big Sur compliant square icon. Please consider this feature! I know some individual users have already created their own versions of square icons here: https://macosicons.com
